webviz_subsurface/plugins/_co2_leakage/_utilities/callbacks.py

```diff
@@ -86,7 +86,9 @@ def from_server(
             visualization_info,
             map_attribute_names,
         )
-        assert surf_meta is not None  # Should not occur
+        if surf_meta is None:  # Surface file does not exist
+            return None, None
```

Comment:
Er det for å håndtere at noen av filene for 2D-kart mangler? Vi har et nytt issue på det i sprint 3, kan være andre ting også, hvis man ikke har noen 2D-kart i det hele tatt.

Reply:
Her var det for å håndtere manglende 2D-kart, ja. Kan se litt grundigere på hva implikasjonene av ingen 2D-kart i det hele tatt blir.

Reply:
Dette issuet i så fall: https://github.com/equinor/ccs-scripts/issues/131
Kan se an litt om hva vi vil støtte, om vi skal kreve at man må ha både kart og CSV-filer, eller kun ha 1 av 2.